prime-router/src/test/kotlin/azure/resources/validationTests/FHIR_to_FHIR/input.fhir

```diff
@@ -0,0 +1,13618 @@
+{
```

Comment:
Why are these under the azure folder when they are used in tests under the fhirengine folder? Seems like a better place would be `src/test/resources`?